src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/01 Executors/05 constant-arrival-rate.md

```diff
@@ -76,7 +76,7 @@ export const options = {
       timeUnit: '1s',
 
       // Pre-allocate VUs
-      preAllocatedVUs: 50,
+      preAllocatedVUs: 2,
```

Comment:
This seems to be aligned with 

> It allocates 50 VUs for k6 to dynamically use as needed.

Can you please expand on what is inconsistent ?